sponsors.html

```diff
@@ -15,154 +15,165 @@
+              <li><a href="about.html">About</a></li>
+              <li class="active"><a href="sponsors.html">Sponsors</a></li>
+            </ul>
           </div>
-        </nav>
+        </div>
+      </nav>
 
-      </div>
     </div>
+  </div>
 
+  <!-- Marketing messaging and featurettes
+  ================================================== -->
+  <!-- Wrap the rest of the page in another container to center all the content. -->
 
-
-    <!-- Marketing messaging and featurettes
-    ================================================== -->
-    <!-- Wrap the rest of the page in another container to center all the content. -->
-
-    <div class="container marketing">
+  <div class="container marketing">
 
 
-    <!-- Three columns of text below the carousel -->
-      <div class="row featurette text-center">
-        <div class="col-lg-12">
-          <h2 class="featurette-heading">Thanks to Our Sponsors!</h2>
-        </div>
+  <!-- Three columns of text below the carousel -->
+    <div class="row featurette text-center">
+      <div class="col-lg-12">
+        <h2 class="featurette-heading">Thanks to Our Sponsors!</h2>
       </div>
+    </div>
+
+  <hr class="featurette-divider">
+
+    <div class="row text-center">
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/ASC Trust.png" alt="Generic placeholder image" width="140">
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/Coca-Cola Guam.png" alt="Generic placeholder image" width="140">
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/Docomo Pacific.png" alt="Generic placeholder image" width="140">
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/GCC.png" alt="Generic placeholder image" width="140">
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/GTA.png" alt="Generic placeholder image" width="140">
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/Hilton Guam.png" alt="Generic placeholder image" width="140">
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/IT_E.png" alt="Generic placeholder image" width="140">
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/Pepsi.png" alt="Generic placeholder image" height="140">
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/UOG.png" alt="Generic placeholder image" width="140">
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-3">
+        <img class="img-rounded" src="logos/sponsors/triplej-logo.jpg" alt="Generic placeholder image" width="140">
```

Comment:
did we double check with Triple J to make sure this is the logo they want to use?